test/smoke/Story.js

```diff
@@ -0,0 +1,17 @@
+const sel = require('../../data/selectors.json');
+const exp = require('../../data/expected.json');
+
+ describe ('Label', function() {
+
+    describe ('Label is displayed', function(){
+
+        it ('TCT-001 Label', function () {
+            browser.url('https://qa-apps.netlify.app/app_my_hero');
+            const story = $(sel.story2).isDisplayed();
```

Comment:
Nasty your test is failed because selector is wrong (you wrote selector wrong and selector sel.story is not for your test ) - the correct selector is **label** from our file with selectors
1. your test (in our test cases file) is not only to verify that is displayed but also verify that the text is correct 
**verify that after the Age section the Story dropdown is present with the text: 4. What type of story would you like to read?**
2. so  
it ('TCT-001 Label', function () {
            browser.url('');
            const story = $$(sel.label)[3].isDisplayed();
            const storyText =$$(sel.label)[3].getAttribute('textContent');
            expect(story).toEqual(true);
           **here please write expect(storyText ).toEqual 4. What type of story would you like to read?** and do not forget to add 4. What type of story would you like to read? in expected file
        });